dashboard/test/factories.rb

```diff
@@ -324,6 +324,16 @@
     status 'present'
   end
 
+  factory :peer_review do
+    user nil
+    from_instructor false
+    script nil
+    level nil
```

Comment:
if you leave something out of a factory the factory will create one for you... that might make it easier to use. It doesn't appear that you are using this factory yet anyway.